server/apiserver/argoserver.go

```diff
@@ -49,6 +49,7 @@ import (
 const (
 	// MaxGRPCMessageSize contains max grpc message size
 	MaxGRPCMessageSize = 100 * 1024 * 1024
+	Localhost = "127.0.0.1"
```

Comment:
Do you really need a constant which is used only on place?  you can do it inline.